fundmanager/fundmanager.go

```diff
@@ -16,19 +16,28 @@ import (
 	"github.com/filecoin-project/lotus/chain/types"
 	"github.com/filecoin-project/specs-actors/actors/builtin/market"
 	"github.com/google/uuid"
+	"github.com/ipfs/go-cid"
 	logging "github.com/ipfs/go-log/v2"
 )
 
 var log = logging.Logger("funds")
 
 type fundManagerAPI interface {
+	MarketAddBalance(ctx context.Context, wallet, addr address.Address, amt types.BigInt) (cid.Cid, error)
 	StateMarketBalance(ctx context.Context, addr address.Address, tsk types.TipSetKey) (api.MarketBalance, error)
 	WalletBalance(context.Context, address.Address) (types.BigInt, error)
 }
 
 type Config struct {
-	EscrowWallet address.Address
+	// The address of the storage miner, used as the target address when
+	// moving funds to escrow
+	StorageMiner address.Address
```

Comment:
Nit: I think we can come up with a better name here. EscrowWallet or EscrowAddr or StorageMinerAddr... not sure right now.